pkgs/applications/version-management/meld/default.nix

```diff
@@ -45,9 +45,12 @@ python3.pkgs.buildPythonApplication rec {
     gtksourceview4
     gsettings-desktop-schemas
     gnome.adwaita-icon-theme
-  ];
+  ] ++ (with python3.pkgs; [
+    pygobject3
+    pycairo
+  ]);
```

Comment:
It appears to work without this?

Reply:
Removed `buildInputs` addition and left only `pythonPath`.